src/Documentation/Markdown/Markdown.js

```diff
@@ -127,94 +128,120 @@ export default class Markdown extends Component {
   }
 
   render() {
-    const { markdown, githubLink, section, file, onFileSelect } = this.props
+    const {
+      markdown,
+      githubLink,
+      section,
+      file,
+      onFileSelect,
+      load
+    } = this.props
     const files = sidebar[section].files
     const fileIndex = files.findIndex(f => f === file)
     const showPrev = fileIndex > 0
     const showNext = fileIndex + 1 < sidebar[section].files.length
 
-    return (
-      <Content>
-        <GithubLink href={githubLink} target="_blank">
-          <i /> Edit on Github
-        </GithubLink>
-        <ReactMarkdown
-          key={`${section}-${fileIndex}`}
-          className="markdown-body"
-          escapeHtml={false}
-          source={markdown}
-          renderers={{
-            code: CodeBlock,
-            heading: HeadingRenderer,
-            virtualHtml: HtmlRenderer
-          }}
-          astPlugins={[linker()]}
-        />
-        <NavigationButtons>
-          <Button
-            onClick={() => onFileSelect(files[fileIndex - 1], section)}
-            disabled={!showPrev}
-          >
-            <i className="prev" />
-            <span>Prev</span>
-          </Button>
-          <Button
-            onClick={() => onFileSelect(files[fileIndex + 1], section)}
-            disabled={!showNext}
+    if (load) {
+      return (
+        <Content>
+          <div
+            style={{
+              width: '100%',
```

Comment:
let's do the style as we do in other cases

Reply:
fix it